### ISSUE TITLE
AUR package build fails using Rust Nightly v1.51.0-nightly

### ISSUE THREAD

**Issue by AkechiShiro** [kdheepak/taskwarrior-tui#196]
**Describe the bug**

Well, it is simple the build fails with an error during building.

**To Reproduce**

- Just run under Archlinux `sudo aura -Axc taskwarrior-tui-git` and type in your password.

**Environment (please complete the following information):**

- Operating System: Linux, distro ArchLinux 
- Installation: Using the AUR (Arch User Repositories) 

**Additional context and information**
Someone has [commented that their build is successful on ArchLinux](https://aur.archlinux.org/packages/taskwarrior-tui-git/#comment-804430) with the exact AUR package of `taskwarrior-tui-git` which seems weird to me, and points me to think that perhaps my actual Rust version could be triggering this issue.

Here is the precise error that is thrown during the building process : 
```rust
error[E0433]: failed to resolve: could not find `addr_of` in `ptr`
   --> /home/zwei/.cargo/registry/src/github.com-1ecc6299db9ec823/anyhow-1.0.40/src/error.rs:606:14
    |
606 |         ptr::addr_of!((*unerased.as_ptr())._object) as *mut E,
    |              ^^^^^^^ could not find `addr_of` in `ptr`

error[E0433]: failed to resolve: could not find `addr_of` in `ptr`
   --> /home/zwei/.cargo/registry/src/github.com-1ecc6299db9ec823/anyhow-1.0.40/src/error.rs:647:22
    |
647 |                 ptr::addr_of!((*unerased.as_ptr())._object) as *mut E,
    |                      ^^^^^^^ could not find `addr_of` in `ptr`

   Compiling futures-lite v1.11.3
error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0433`.
error: could not compile `anyhow`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: build failed
==> ERROR: A failure occurred in build().
    Aborting...
aura >>= There was a makepkg failure.
aura >>= Would you like to continue anyway? [Y/n] y
aura >>= Every package failed to build.
```

**Comment by kdheepak** [comment]
It is very likely the rust compiler version. What version are you using? Can you update to the latest?

**Comment by kdheepak** [comment]
Oh you have it there in the title. `1.51.0-nightly`. Let me check what I’m using on my macOS machine.

**Comment by AkechiShiro** [comment]
You were correct, updating the rust compiler fixed the issue, I updated to `v1.53.0-nightly` and rebuild, issue is gone.

**Comment by kdheepak** [comment]
I’ll add a note in the README about the version of rust compiler I’m using.
